src/vmm/src/persist.rs

```diff
@@ -48,39 +47,18 @@ use crate::{mem_size_mib, vstate, EventManager, Vmm, VmmError};
 const FC_V0_23_MAX_DEVICES: u32 = 11;
```

Comment:
I think this constant can go away :thinking:

Reply:
yup!